Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs

```diff
@@ -607,17 +645,41 @@ private static void AddSubmissionTypes(Problem problem, AdminActionContext actio
     {
         var newSubmissionTypes = actionContext.GetSubmissionTypes()
             .Where(x => x.IsChecked)
-            .Select(x => new SubmissionTypeInProblem
+            .Select(x =>
             {
-                ProblemId = problem.Id,
-                SubmissionTypeId = int.Parse(x.Value!.ToString() !),
-                SolutionSkeleton = x.Expand != null && x.Expand.Value != null
-                    ? x.Expand.Value!.ToString() !.Compress()
-                    : Array.Empty<byte>(),
+                var psted = new ProblemSubmissionTypeExecutionDetails();
```

Comment:
We do not use short names/abbreviations. You can just use `submissionTypeDetails` for a variable name

Reply:
Done.